docs/user_manual/getting_verified.md

```diff
@@ -32,23 +32,21 @@ because that server is where most game servers are listed.
 Getting registered for the login server run by Griffon26 is an automated process:
 
 1. First log in to the Hirez servers
+1. Join a game server
+1. Open the Chat Console. (This might not have a keybinding, add one through `Settings > Keybindings > Interface`). Note: This console is different to the Unreal Console (~)
```

Comment:
I think it's possible for anyone to open the chat console by pressing ESC, followed by t (assuming you haven't changed your chat key).